File: app/src/main/java/com/zaf/econnecto/ui/presenters/operations/IMyBusinessLatest.java
package com.zaf.econnecto.ui.presenters.operations;


import com.zaf.econnecto.network_call.response_model.img_data.ViewImageData;
import com.zaf.econnecto.network_call.response_model.my_business.BasicDetailsResponse;

import org.jetbrains.annotations.NotNull;

import java.util.List;

public interface IMyBusinessLatest {

    void updateBannerImage(@NotNull List<? extends ViewImageData> data);
    void updateBasicDetails(BasicDetailsResponse basicDetailsResponse, boolean imageUpdate);
}

File: app/src/main/java/com/zaf/econnecto/ui/presenters/operations/IMyBusinessLatest.kt
package com.zaf.econnecto.ui.presenters.operations

import com.zaf.econnecto.network_call.response_model.my_business.BasicDetailsResponse
import com.zaf.econnecto.ui.activities.mybiz.*

interface IMyBusinessLatest {

    fun updateBasicDetails(basicDetailsResponse: BasicDetailsResponse, imageUpdate: Boolean)
    fun updateOperatingHours(data: OPHoursData?)
    fun updateProductServiceSection(data: List<ProductNServiceData>?)
    fun updateBrochureSection(data: List<BrochureData>?)
    fun updateAmenitiesSection(data: List<AmenityData>?)
    fun updatePaymentSection(data: List<PaymentMethodData>?)
    fun updatePricingSection(data: List<PricingData>?)
    fun updateCategories(data: List<UserCategoryData>?)
}